### ISSUE TITLE
Build not success with Qt 5.11.1

### ISSUE THREAD

**Issue by neoz** [fpoussin/Qt5-MSVC-Static#6]
qmachparser.obj : warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operation that consumes this library
qelfparser_p.obj : warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operation that consumes this library
jom: Z:\Working\Qt\Qt5-MSVC-Static\sources\qtbase-everywhere-src-5.11.1\build\Makefile [sub-src-make_first] Error 2

**Comment by fpoussin** [comment]
Can you try a manual build to confirm it's actually caused by the tools and not your setup?
Thanks

**Comment by fpoussin** [comment]
Please reopen if needed

**Comment by psadaic** [comment]
jom: C:\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\build\src\angle\src\libGLESv2\Makefile.Release [.obj\release\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\src\3rdparty\angle\src\third_party\murmurhash\MurmurHash3.obj] Error 1
jom: C:\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\build\src\angle\src\libGLESv2\Makefile [release] Error 2
jom: C:\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\build\src\angle\src\Makefile [sub-libGLESv2-make_first-ordered] Error 2
jom: C:\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\build\src\angle\Makefile [sub-src-make_first] Error 2
jom: C:\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\build\src\Makefile [sub-angle-make_first] Error 2
        lib /NOLOGO  /OUT:..\..\..\lib\qtpcre2.lib @C:\Users\Patrik\AppData\Local\Temp\qtpcre2.lib.12412.44781.jom
jom: C:\Users\Patrik\Documents\Qt5-MSVC-Static-5111\sources\qtbase-everywhere-src-5.11.1\build\Makefile [sub-src-make_first] Error 2

Yes it fails while building qt 5.11.1. There isn't much output but seems like if fails while compiling some of the angle libraries.

**Comment by fpoussin** [comment]
Did you run this build manually as well to confirm it's induced by the tools?

**Comment by psadaic** [comment]
Yes manual build from qt source completed without any issues with same qt configuration. I did try to use new gnuwin32/bin files considering my issue was with angle since it is stated in build docs that GPerf, Bison and Flex libs are used during angle build which have been updated since the last qt version.

**Comment by fpoussin** [comment]
Ok I'll have a look, thanks!

**Comment by fpoussin** [comment]
@psadaic can you paste the configuration parameters so I can compare?
Thanks

**Comment by psadaic** [comment]
Sure.

`configure -opensource -release -confirm-license -opengl dynamic -mp -static -static-runtime -no-shared -qt-libpng -qt-libjpeg -qt-zlib -qt-pcre -no-compile-examples -nomake examples -no-icu -optimize-size`

**Comment by psadaic** [comment]
Managed to identify the problem by trying to compile with nmake directly, jom wasn't giving the actual error message.

`murmurhash\MurmurHash3.cpp : fatal error C1083: Cannot open compiler generated file: '': Invalid argument`

Error occured due to too long path. CL is by default compiled with MAX_PATH=260 flag and therefore caused this error since my build folder was already nested under few folder levels. Solution for this might be doing some wrapper script to eliminate this issue but for now my quick solution is just to use shorter build path.

As for issues @neoz posted I think I've noticed them while trying to make debug qt build even on previous qt release (5.10.1). I do not use static debug builds so didn't go too deep into that but might be helpful for you.

**Comment by fpoussin** [comment]
Seems like this is completely unrelated to the original issue.
Which OS / Filesystem / Compiler do you use?

Thanks

**Comment by fpoussin** [comment]
No worries, glad you found out what was wrong.
